src/logger/loggerConfig.c

```diff
@@ -197,6 +197,12 @@ static void resetCanConfig(CANConfig *cfg)
     }
 }
 
+uint8_t filter_can_channel(uint8_t value)
+{
+        if (value >= CONFIG_CAN_CHANNELS)
```

Comment:
thanks for the min() suggestion. 

I thought about the 2nd part. We have a theme throughout where we filter input values to guarantee things are in range, so I was emulating this. Part of the goal was to be able to easily import CAN mappings from a system that supports 2 CAN channels into systems that support 1.